File: Frontend/Private/gco-private/src/app/model/core/team-scoreboard.ts
import { AgeCategory } from './age-category'
import { Apparatus } from './apparatus'
import { TeamScoreboardSlot } from './team-scoreboard-slot';

export interface TeamScoreboard{
    id: string;
    competitionId: string;
    ageCategory: string;
    apparatuses: Apparatus[];
    slots: TeamScoreboardSlot[];
}

File: Backend/scoring_service/core/domain/team_scoreboard.go
package domain

import "github.com/google/uuid"

type TeamScoreboard struct {
	ID            uuid.UUID
	CompetitionID uuid.UUID
	Competition   Competition
	AgeCategory   string
	Apparatuses   []Apparatus `gorm:"serializer:json"`
	Slots         []TeamScoreboardSlot
}
